sdk/lib/cpprt/__cxa_pure_virtual.cpp

```diff
@@ -0,0 +1,13 @@
+/*
+ * PROJECT:     GCC c++ support library
```

Comment:
```suggestion
 * PROJECT:     GCC C++ support library
```
There are differences between headers of the files in this directory though.